### PR TITLE
Added new blog post about Haiku survey

### PR DESCRIPTION
**Pull request by jt15s**
This is a new pull request using a new branch and after rebasing the master branch of my fork of the Haiku website, as suggested by @leavengood. Please let me know if any modifications are needed. 

Thanks!

### CODE REVIEW

File: content/blog/jt15s/2021-03-04_haiku_survey_februarymarch_2021.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title = "Haiku Survey February/March 2021"
+author = "jt15s"
+date = "2021-03-04 19:54:14+11:00"
+tags = ["haiku", "software"]
++++
+
+Hi all, hope you are all having a great start to March! kallisti5 has put together a survey to help determine the priorities the Project should focus on for the immediate future. If you haven't already, we would really appreciate if you filled out the survey - it is 100% anonymous and you do not have to enter any personal details.
+[**You can fill out the survey here.**](https://forms.gle/A3GQxUu6pHsjkZH87)
+
+Please do not hesitate to give your feedback on Haiku in the survey - your feedback will influence any decisions we make in the future. Additionally, if you have any feedback on how the survey should be conducted, please let kallist5 know either through the forums or on the Haiku mailing lists.
```
Comment (leavengood) [inline content/blog/jt15s/2021-03-04_haiku_survey_februarymarch_2021.md:12]:
```suggestion
Please do not hesitate to give your feedback on Haiku in the survey - your feedback will influence any decisions we make in the future. Additionally, if you have any feedback on how the survey should be conducted, please let kallisti5 know either through the forums or on the Haiku mailing lists. 
```

**Comment by leavengood** [comment]
Thanks for the rebase. Git can be "fun", but is a useful tool to know. Just found this one small typo.

File: content/blog/jt15s/2021-03-04_haiku_survey_februarymarch_2021.md (hunk 1 above)
Comment (humdingerb) [inline content/blog/jt15s/2021-03-04_haiku_survey_februarymarch_2021.md:12]: I wouldn't sign away all future decisions to that survey... :)

```suggestion
Please do not hesitate to give your feedback on Haiku in the survey - your feedback can influence decisions we make in the future. Additionally, if you have any feedback on how the survey should be conducted, please let kallisti5 know either through the forums or on the Haiku mailing lists. 
```

**Comment by jt15s** [comment]
Thanks for that @leavengood and @humdingerb - for some reason Github wouldn't let me commit the second suggestion straight away so I had to do it manually.

**Pull request merged**